pkg/settings/setting.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+const (
+	RancherVersionDev = "2.6.99"
```

Comment:
I'm not sure, @kinarashah might now. This was the default the kdm package had so I left it as is.

Reply:
I think we need to understand the meaning of this number.  If it changes we should also document it as part of the release process or ideally automate it.

Reply:
I agree. However, I only moved the GetRancherVersion function to a different package to fix cyclic import issues, and didn't change the logic of the function itself so it should be discussed and implemented in a separate issue to avoid creeping into the scope of this issue.

Reply:
My 2 cents on this. `dev` is still the default value for [ServerVersion](https://github.com/rancher/rancher/blob/c1cb14552198518f759ca3b8d41ea7edb13f67ec/pkg/settings/setting.go#L67) setting. `GetRancherVersion()` is just transforming this if required, to semver format to allow charts filtering by Rancher version. The logic seems the same.

Reply:
@rawmind0 That's what I thought too.

Reply:
@rawmind0 is exactly right. Is there no way to continue doing this with KDM? Adding another constant that needs to be maintained depending on version should be avoided. KDM needs to do it anyway and it makes sense to use it from whichever KDM version is being imported.

Reply:
To me it makes more sense for it to live in the settings package since all it does is read the settings, but even if we decide to keep it in KDM, I can't import it into this package and other packages that use the GetImage function due to cyclic import issues (Jacob explains it in this [issue](https://github.com/rancherlabs/engineering-improvements/issues/9)). In this package I could pass the rancher version we get fromt he TAG around to make it work, but then the catalog package will need to pass in a rancher version as an argument and you can't import KDM there either to get it.